web/html/src/manager/minion/details/support-data.renderer.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from "react";
+
+import SpaRenderer from "core/spa/spa-renderer";
+
+import { SupportData, UploadRegion } from "./support-data";
+
+export const renderer = (
```

Comment:
Renderers in the rest of the codebase usually (although unfortunately not always) follow the pattern of taking two params, `id: string` and then an object with the props. It would be preferable to follow this, as it will make future changes we have planned around Jade renderers easier.

Reply:
Fixed, I introduces a props object.